src/app/shared/navbar/navbar.component.html

```diff
@@ -19,36 +22,36 @@
 
       <ng-template #userTemplate>
         <ng-container *ngIf="loggedin">
-          <div class="contributor" id="contributor-link" #contributorDropdownButtonElement (click)="contributorDropdown(true)"><span>Contribute</span> <i class="fas fa-chevron-down"></i></div>
-          <div class="right library" id="library-link" routerLink="/library"></div>
-          <div id="clark-user-dropdown" #userDropdownButtonElement class="left flex h flex-end user-menu" (click)="userDropdown(true)">
+          <button class="contributor" id="contributor-link" #contributorDropdownButtonElement (click)="contributorDropdown(true)"><span>Contribute</span> <i class="fas fa-chevron-down"></i></button>
+          <a class="right library" id="library-link" routerLink="/library"></a>
+          <button id="clark-user-dropdown" #userDropdownButtonElement class="user" (click)="userDropdown(true)">
             <div class="navbar-gravatar" id="gravatar" [ngStyle]="{'background-image': 'url(' + gravatarImage(200) + ')'}"></div>
             <i class="fas fa-chevron-down"></i>
-          </div>
+          </button>
 
           <clark-context-menu *ngIf="contributorDown" [anchor]="contributorDropdownButtonElement" [offset]="{ top: 10, left: 5 }" (close)="contributorDropdown()">
             <div #contextMenu>
               <ul>
-                <li routerLink="/onion/dashboard" (click)="contributorDropdown()"><i class="far fa-list"></i>Your Dashboard</li>
-                <li routerLink="/onion/learning-object-builder" (click)="contributorDropdown()"><i class="far fa-plus"></i>Create a Learning Object</li>
+                <li tabindex="0" routerLink="/onion/dashboard" (click)="contributorDropdown()" id="contributorDropdown1"><i class="far fa-list"></i>Your Dashboard</li>
```

Comment:
Could we potentially get a better ID here?
![image](https://user-images.githubusercontent.com/7431390/61003682-82d1f180-a332-11e9-8197-b5f958f35a41.png)

Reply:
That has to go actually. Remnant of something I tried for the search